docs/pages/management/admin/labels.mdx

```diff
@@ -1,405 +1,323 @@
-|[Windows desktops](../../desktop-access/getting-started.mdx)|Dynamic labels are not supported for Windows desktops. See our [Desktop Access documentation](../../desktop-access/rbac.mdx#labeling) for how to label desktops.|
+1. (!docs/pages/includes/install-linux.mdx!)
 
-</Admonition>
+1. Generate an invitation token for your local computer.
+   
+   The invitation token is required for the local computer to join the Teleport cluster.   
+   The following example generates a new token that is valid for five minutes and can be used 
+   to enrolled a server, database, and application resource:
+   
+   ```code
+   $ tctl tokens add --ttl=5m --type=node,db,app
+   # The invite token: (=presets.tokens.first=)
+   ```
 
-## Step 1/3. Prepare your Node
+1. List all generated non-expired tokens by running the following command:
+   
+   ```code
+   $ tctl tokens ls
+   # Token                            Type        Labels Expiry Time (UTC)               
+   # -------------------------------- ----------- ------ ------------------------------- 
+   # (=presets.tokens.first=)         Node,Db,App        10 Aug 23 19:49 UTC (4m11s) 
+   ```
 
-### Install Teleport on your Node
+1. Copy the token and assign it to an environment variable on the computer you are enrolling 
+as a resource:
+   
+   ```code
+   $ export INVITE_TOKEN=<token>
+   ```
 
-On the host where you will run your Teleport Node, follow the instructions for
-your environment to install Teleport.
+## Step 2/3. Apply a static label
 
-(!docs/pages/includes/install-linux.mdx!)
-
-### Generate a token
-
-On your local machine, use the `tctl` tool to generate an invite token that your
-Node will use to join the cluster. In the following example, a new token is
-created with a TTL of five minutes:
-
-```code
-# Generate a short-lived invitation token for a new node:
-$ tctl tokens add --ttl=5m --type=node
-# The invite token: (=presets.tokens.first=)
-
-# You can also list all generated non-expired tokens:
-$ tctl tokens ls
-# Token                            Type            Expiry Time
-# ------------------------         -----------     ---------------
-# (=presets.tokens.first=)         Node            25 Sep 18 00:21 UTC
-```
-
-Copy the token. On the remote host that you will add to your Teleport cluster as a
-Node, assign the token to an environment variable:
-
-```code
-$ export INVITE_TOKEN=<token>
-```
-
-We will use this in the next step.
-
-## Step 2/3. Apply a static label to your Node
-
-You can configure Node labels by editing Teleport's configuration file, then
+You can configure static labels by editing the Teleport configuration file, then
 starting Teleport.
 
-On the host where you will run your Node, paste the following content into
-`/etc/teleport.yaml` and adjust it for your environment:
-
-<ScopedBlock scope={["oss", "enterprise"]}>
+To add a static label:
+
+1. Open the Teleport configuration file—by default, `/etc/teleport.yaml`—in an editor on the computer 
```

Comment:
```suggestion
1. Open the Teleport configuration file — by default, `/etc/teleport.yaml`—in an editor on the computer 
```

Maybe? I read this as "configuration file-by" and didn't know what that phrase meant.

Also, two hyphens in such a short sentence seems excessive, no?

Reply:
It's an em dash and can be quite effective in setting out text without disrupting sentence flow. Typically preferred over using parentheses.